RaveBusinessLogic/Confinement.xml

```diff
@@ -12,21 +12,21 @@
           <Node xpathlabel="Name" xpath="Outputs/Geopackage/Layers/Vector[@id='CONFINEMENT_MARGINS']" type="line" id="confinement_margins" symbology="confinement_margins" />
           <Node xpathlabel="Name" xpath="Outputs/Geopackage/Layers/Vector[@id='CONFINEMENT_RATIO']" type="line" id="confinement_ratio" symbology="confinement_ratio" />
           <Node label="Constriction Ratio" xpath="Outputs/Geopackage/Layers/Vector[@id='CONFINEMENT_RATIO']" type="line" id="constriction_ratio" symbology="constriction_ratio" />

```

Comment:
So is this actually a "Constriction Ratio"? The xpath says "Confinement Ratio"? Two problems here.
- [ ] Make sure this is actually coming from the "Constriction_Ratio" field in attribute table (we know it is from [here](https://github.com/Riverscapes/RiverscapesXML/blob/confinement_project/Symbology/qgis/Confinement/constriction_ratio.qml#L15).
- [ ] Label and Display as a 0 - 1 value (not a binary) as "Constriction Proportion". 

Constriction Ratio in the literature often refers to a discrete constriction point (e.g. a debris fan rapid) and is a ratio of upstream or downstream channel width to width at constriction. We proposed in paper Constriction proportion as something analogous to the confinement ratio, in that it is the percentage of that reach length that is confined on both sides. See [this issue](https://github.com/Riverscapes/riverscapes-tools/issues/316)